Source/Core/Core/HW/EXI/EXI_DeviceMemoryCard.cpp

```diff
@@ -272,13 +267,14 @@ void CEXIMemoryCard::CmdDone()
 void CEXIMemoryCard::TransferComplete()
 {
   // Transfer complete, send interrupt
-  ExpansionInterface::GetChannel(m_card_index)->SendTransferComplete();
+  ExpansionInterface::GetChannel(ExpansionInterface::SlotToEXIChannel(m_card_slot))
+      ->SendTransferComplete();
 }
 
 void CEXIMemoryCard::CmdDoneLater(u64 cycles)
 {
-  CoreTiming::RemoveEvent(s_et_cmd_done[m_card_index]);
-  CoreTiming::ScheduleEvent(cycles, s_et_cmd_done[m_card_index], m_card_index);
+  CoreTiming::RemoveEvent(s_et_cmd_done[m_card_slot]);
+  CoreTiming::ScheduleEvent(cycles, s_et_cmd_done[m_card_slot], static_cast<int>(m_card_slot));
```

Comment:
The `userdata` parameter here is an `u64`, so might want to cast to that instead.